submission_runner.py

```diff
@@ -8,7 +8,8 @@
     --submission_path=reference_submissions/mnist/mnist_jax/submission.py \
     --tuning_ruleset=external \
     --tuning_search_space=reference_submissions/mnist/tuning_search_space.json \
-    --num_tuning_trials=3
+    --num_tuning_trials=3 \
+    --experiment_dir=experiments
```

Comment:
I would make the example have an absolute dir, even if people need to change it, so something like `/home/pomonam/mlcommons`

that way there isn't any confusion about where the dir is relative to